src/index.html

```diff
@@ -7,8 +7,30 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo-link">
+        <img src="images/logo.png" alt="Moyo Logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="list_item"><a href="#" class="link is-active">Apple</a></li>
+          <li  class="list_item"><a href="#" class="link">Samsung</a></li>
+          <li class="list_item"><a href="#" class="link">Smartphones</a></li>
+          <li class="list_item">
+            <a href="#" class="link"
+            data-qa="hover">Laptops & Computers</a>
+          </li>
+          <li class="list_item"><a href="#" class="link">Gadgets</a></li>
+          <li class="list_item"><a href="#" class="link">Tablets</a></li>
+          <li class="list_item"><a href="#" class="link">Photo</a></li>
+          <li class="list_item"><a href="#" class="link">Video</a></li>
```

Comment:
```suggestion
          <li class="list_item">
            <a href="#" class="link is-active">Apple</a>
          </li>
          <li  class="list_item">
            <a href="#" class="link">Samsung</a>
          </li>
          <li class="list_item">
            <a href="#" class="link">Smartphones</a>
          </li>
          <li class="list_item">
            <a 
              href="#" 
              class="link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
          </li>
          <li class="list_item">
            <a href="#" class="link">Gadgets</a>
          </li>
          <li class="list_item">
            <a href="#" class="link">Tablets</a>
          </li>
          <li class="list_item">
            <a href="#" class="link">Photo</a>
          </li>
          <li class="list_item">
            <a href="#" class="link">Video</a>
          </li>
```
Consider fixing indents according to the code style guide